doc/Makefile

```diff
@@ -59,6 +59,8 @@ run: install
 # Doesn't depend on $(BUILDDIR) to rebuild properly at every run.
 html: install
 	. $(VENV); $(SPHINXBUILD) -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" -w .sphinx/warnings.txt $(SPHINXOPTS)
+	@echo -e "\nClipping 'How to' from Table of Contents:"
+	@sed -i '/toctree-l2.*>How to / s/How to \(.\)/\U\1/' _build/**/*.html _build/*.html
```

Comment:
This seems duplicated with `howto.sh`, can we re-use the script here, so we don't define the same logic in two places?

Reply:
> The first commit seems to be a left-over from the rebase.

Right. I'll squash them.

> It's still here :)

Aargh. It came back wirth the rebase.

> This seems duplicated with howto.sh, can we re-use the script here, so we don't define the same logic in two places?

OK

Reply:
> This seems duplicated with `howto.sh`, can we re-use the script here, so we don't define the same logic in two places?

I replaced the `sed` command (and the `howtos.sh` script) with a bit of Python code added to `conf.py`. That way, it's called as a part of the Sphinx build both on local and on RTD, and it also works when `make run` (.e. `sphinx-autobuild`) is used.